Rb.Forms.Barcode.Droid/Camera/CameraConfigurator.cs

```diff
@@ -158,6 +180,70 @@ private void invokeIfAvailable(string value, Action<string> cb)
             cb.Invoke(value);
         }
 
+        private AndroidCamera.Size determinePreviewResolution(AndroidCamera.Parameters parameters)
+        {
+            var referenceRatio = getViewRatio();
+
+            /**
+             * The aspect ratio between the preview resolution and the actual preview view
+             * is not allowed to expand the configurated aspect ratio threshold.
+             */
+            Func<AndroidCamera.Size, bool> ratioFilter = (r) =>  {
```

Comment:
Will do the better naming, but i dont understand the first comment regarding "that i only need r" :(